sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  TextAnalysisAuthoringDeployProjectParameters,
+  TextAnalysisAuthoringTrainBodyParam,
+  getLongRunningPoller,
+  CreateProjectOptions,
+  TextAuthoringClient,
+  ExportedCustomSingleLabelClassificationProjectAssets,
+  ExportedCustomMultiLabelClassificationProjectAssets,
+  ExportedCustomEntityRecognitionProjectAssets,
+} from "@azure/ai-language-textauthoring";
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+import { DefaultAzureCredential } from "@azure/identity";
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+import path from "path";
+import decompress from "decompress";
+import {
+  customEntityAssets,
+  customMultiLabelAssets,
+  customSingleLabelAssets,
+} from "../customTestsAssets";
+dotenv.config();
```

Comment:
This wont work in the browser, I think we will need to access `env` exported by the recorder.